cfgov/paying_for_college/models/search.py

```diff
@@ -0,0 +1,21 @@
+from paying_for_college.documents import SchoolDocument
+from search.elasticsearch_helpers import make_safe
+
+
+class SchoolSearch:
+    def __init__(self, search_term):
+        self.search_term = make_safe(search_term).strip()
+        self.results = []
+
+    def autocomplete(self):
+        if self.search_term != '':
+            search = SchoolDocument.search().query(
+                'match', autocomplete=self.search_term)
+            total_results = search.count()
+            search = search[0:total_results]
+            response = search.execute()
+            self.results = response[0:total_results]
```

Comment:
Why are we still capturing a count and delivering the entire response set? For autocomplete, this delivers tons of unhelpful partial matches. The default top 10 objects is a better starting point.

```suggestion
            response = search.execute()
            self.results = response
```

Reply:
Modified autocomplete to return the default top 10 hits instead of the entire response set

Reply:
Agreed, the only reason to do a count is to get all results, something like autocomplete is better off returning the highest 10 matches.